Assets/LeapMotion/Editor/CustomEditorBase.cs

```diff
@@ -118,6 +119,19 @@ public class CustomEditorBase : Editor {
       }
     }
 
+    /// <summary>
+    /// Defer rendering of a property until the end of the inspector.  Deferred properties
+    /// are drawn in the REVERSE order they are deferred!  NOT by the order they appear in the 
```

Comment:
The reasoning is to deal with tall hierarchies of objects that all want to specify deferred properties.  Imagine we have class A, B and C.  C derives from B, and B derives from A.  A defers properties A1 and A2, B defers properties B3 and B4, and C defers properties C5 and C6.

If we drew properties in order, we would see:
```
normal properties of A
normal properties of B
normal properties of C
A1
A2
B3
B4
C5
C6
```

However I think that it would be much nicer if we saw this instead:
```
normal properties of A
normal properties of B
normal properties of C
C5
C6
B3
B4
A1
A2
```

Kind of an outside-in approach.  Class A defines what comes first and last and leaves a space in between for deriving classes.